frontend/src/libs/helpers/get-formatted-time.helper.ts

```diff
@@ -0,0 +1,13 @@
+import { format } from 'date-fns';
+
+const EPOCH_MILLISECONDS = 0;
+const TIME_FORMAT = 'mm:ss';
+
+const getFormattedTime = (time: number): string => {
+  const timeDate = new Date(EPOCH_MILLISECONDS);
+  timeDate.setSeconds(time);
```

Comment:
Won't this do the same?
```suggestion
  const timeDate = new Date(time);
```

Reply:
no, because time I receive in seconds, but Date expects to receive milliseconds